Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/CertificateStateHelper.kt

```diff
@@ -15,11 +15,13 @@ import de.rki.coronawarnapp.util.TimeAndDateExtensions.toLocalDateTimeUserTz
 import de.rki.coronawarnapp.util.TimeAndDateExtensions.toShortDayFormat
 import de.rki.coronawarnapp.util.TimeAndDateExtensions.toShortTimeFormat
 
+@Suppress("ComplexMethod")
 fun IncludeCertificateQrcodeCardBinding.bindValidityViews(
     certificate: CwaCovidCertificate,
     isPersonOverview: Boolean = false,
     isPersonDetails: Boolean = false,
-    isCertificateDetails: Boolean = false
+    isCertificateDetails: Boolean = false,
+    badgeCount: Int? = null
```

Comment:
why `nullable` ? it either `0`, or more

Reply:
Because we call `bindValidityViews` from different places where we don't pass any value to the badge count. I can make it not nullable and give it a default value of '0' instead. But it would be the same.